test/passing/object.ml

```diff
@@ -30,7 +30,8 @@ let _ =
 
     constraint t = 'a t
 
-    [%%ext salut, "hello"]
+    [%%ext
+    salut, "hello"]
```

Comment:
Why this change ? It's not related to comments.

Reply:
It was due to the added hvbox around the structure_item and the comments after, sometimes the break was expected for the extensions, even if it fitted on one line (but not always).
I finally put the hvbox only if there are comments after, to reduce the diff.